profiles/nextstrain-public/rename.smk

```diff
@@ -0,0 +1,47 @@
+
+rule all_public:
+    input:
+        [
+            "auspice_renamed/" + build.get("auspice_name", f"{build_name}_{{segment}}").format(segment=segment) + suffix + ".json"
+            for build_name, build in config["builds"].items()
+            for segment in config["segments"]
+            for suffix in ["", "_root-sequence", "_tip-frequencies"]
+        ],
+
+def _get_file_by_auspice_name(wildcards):
+    for build_name, build_params in config["builds"].items():
+        for segment in config["segments"]:
+            if build_params.get("auspice_name", f"{build_name}_{{segment}}").format(segment=segment) == wildcards.auspice_name:
+                return f"auspice/{build_name}_{segment}.json"
+
+    return ""
+
+rule rename_auspice_main:
+    input:
+        _get_file_by_auspice_name,
+    output:
+        "auspice_renamed/{auspice_name}.json",
+    shell:
+        """
+        cp {input} {output}
+        """
+
+rule rename_auspice_root_sequence:
+    input:
+
```

Comment:
Missing input 👀

Reply:
Holy cow! That's an end-of-the-day-emacs error if I ever saw one. Fixed now.